packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -102,6 +102,8 @@ const TSErrors = makeErrorTemplates(
       "'override' modifier cannot appear on an index signature.",
     IndexSignatureHasStatic:
       "Index signatures cannot have the 'static' modifier.",
+    InvalidInterfaceDeclarationInExportDefault:
```

Comment:
I would _slightly_ prefer to name this `MissingExportDefaultInterfaceName` (or even just `MissingInterfaceName` so that we can potentially reuse it for `interface {}` outside of an export default declaration).

The error message is ok.